otelcollector/build/windows/scripts/setup.ps1

```diff
@@ -59,7 +59,7 @@ Write-Host ('Finished Installing Visual C++ Redistributable Package')
 Write-Host ('Installing Telegraf');
 try {
     # Keep version in sync with linux in setup.sh file
-    $telegrafUri = 'https://dl.influxdata.com/telegraf/releases/telegraf-1.23.4_windows_amd64.zip'
```

Comment:
Please upgrade ME too on line 18. Or is it using the latest version already?

Reply:
Yes it is already using the latest version.

Reply:
MdmMetricsExtension.2.2022.1201.1140.nupkg is the version being used here. The latest version is : 2.2023.224.2214

Reply:
Updated